website/src/components/canvas/Sidebar.jsx

```diff
@@ -89,7 +112,18 @@ class Sidebar extends React.Component {
   mouseEnter() {
     this.setState({opacity: 1})
   }
-
+  onClickClose() {
+   this.setState(prevState => ({
+     isOpen: !prevState.isOpen,
+     closed: 1
+   }));
+ }
+ onClickOpen() {
```

Comment:
I wonder if onClickOpen and onClickClose should be renamed or rewritten, because they're not actually doing what the name claims.
For example, rather than setting isOpen to true in onClickOpen, you're simply setting it to the previous state. If for whatever reason it glitched and used a previous state of open, then this handler would close the sidebar instead. Does that make sense? 
I think it would be fine and make your code more readable / logical if you just explicitly set isOpen to true or false as we do with modals and such (see ExportModal.jsx as an example)